compiler/build_cf_graph.jou

```diff
@@ -554,25 +552,25 @@ def build_function_or_method_call(
 
     assert sig != NULL
 
-    args: LocalVariable** = calloc(call->nargs + 2, sizeof(args[0]))
-    k = 0
+    args: LocalVariable** = malloc(sizeof(args[0]) * (call->nargs + 2))  # +1 for self, another because why not
```

Comment:
Might as well add args directly to the instruction instead of building a temporary array.